src/contracts/oracle.js

```diff
@@ -20,7 +18,7 @@ const Oracle = {
       return;
     }
 
-    const oracle = getOracleContract(oracleType, contractAddress);
+    const oracle = getContract(oracleType, contractAddress);
     return await oracle.send('invalidateOracle', {
       methodArgs: [],
       gasLimit: 3000000,
```

Comment:
@dwalintukan `gasLimit` should be defined as a constant

Reply:
added constants for gas limit for all specified calls.